src/include/migraphx/op/concat.hpp

```diff
@@ -18,7 +18,7 @@ namespace op {
 
 struct concat
 {
-    std::size_t axis = 0;
+    int axis = 0;
```

Comment:
Do we usually make these attributes `int64_t`?

Reply:
yes, normally they are int64 if the axis is negative.

Reply:
changed